TF_Settings_Web/src/TouchFree/Connection/MessageReceiver.ts

```diff
@@ -191,7 +198,16 @@ export class MessageReceiver {
         let serviceStatus: ServiceStatus | undefined = this.serviceStatusQueue.shift();
 
         if (serviceStatus !== undefined) {
-            MessageReceiver.HandleCallbackList(serviceStatus, this.serviceStatusCallbacks);
+            const callbackResult = MessageReceiver.HandleCallbackList(serviceStatus, this.serviceStatusCallbacks);
+
+            // If callback didn't happen for known reasons, we cna be sure it's an independent status event rather than a request response
+            if (callbackResult === 'NoCallbacksFound' || callbackResult === 'CallbacksUndefined')
+            {
+                console.log(serviceStatus)
```

Comment:
This console log was useful for testing this but should probably be removed now?